docs/src/examples/components/Tooltip/Variations/TooltipExamplePosition.shorthand.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Button, Grid, Tooltip, Alignment, Position } from '@stardust-ui/react'
+import { useSelectKnob } from '@stardust-ui/docs-components'
+
+const TooltipExamplePosition = () => {
+  const [positionAndAlign] = useSelectKnob({
+    name: 'position-align-s',
```

Comment:
```suggestion
    name: 'position-align',
```

After #1489 will be merged